docs/manual/database/create.md

```diff
@@ -156,75 +154,26 @@ HashMap[] docs = new HashMap[2];
 docs[0] = document;
 docs[1] = document2;
 db.insert("books").docs(docs).apply(new Utils.ResponseListener() {
-    @Override
```

Comment:
Why did you remove this?

Reply:
Seems like an error. Instead pass a handler sort of a thing to the apply function

Reply:
Do this in all similar occurrences

Reply:
How is this?:  
`.apply(myResponseListener);`

Reply:
Perfect. We should do that

Reply:
Cool